test/hotspot/jtreg/serviceability/monitoring/ThreadInfo/getLockOwnerName/getLockOwnerName.java

```diff
@@ -0,0 +1,415 @@
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 8167108 8265153
+ * @summary The test checks that ThreadInfo.getLockOwnerName() returns a
+ *   non-null string for a blocked thread and then makes repeated calls
+ *   to getThreadInfo() and ThreadInfo.getLockOwnerName() until the thread
+ *   has exited.
+ * @requires vm.jvmti
+ * @run main/othervm/native -agentlib:getLockOwnerName getLockOwnerName
+ */
+
+import java.io.PrintStream;
+import java.lang.management.ManagementFactory;
+import java.lang.management.ThreadInfo;
+import java.lang.management.ThreadMXBean;
+
+//
+// main               blocker           contender            releaser
+// =================  ================  ===================  ================
+// launch blocker
+// <launch returns>   blocker running
+// launch contender   enter threadLock
+// <launch returns>   wait for notify   contender running
+// launch releaser    :                 block on threadLock
+// <launch returns>   :                 :                    releaser running
+// wait for           :                 :                    wait for notify
+//  contended enter   :                 :                    :
+// <ready to test>    :                 :                    :
+// getThreadInfo      :                 :                    :
+// verify contender   :                 :                    :
+//  lock owner name   :                 :                    :
+//  is "blocker"      :                 :                    :
+// notify releaser    :                 :                    :
+// loop until thread  :                 :                    wait finishes
+//  is NULL           :                 :                    notify blocker
+//   get thread info  wait finishes     :                    releaser exits
+//   get lock owner   exit threadLock   :                    :
+//    name            blocker exits     enter threadLock     :
+// join releaser      :                 exit threadLock      <exit finishes>
+// <join returns>     :                 contender exits
+// join blocker       <exit finishes>   :
+// <join returns>                       :
+// join contender                       <exit finishes>
+// <join returns>
+//
+
+public class getLockOwnerName {
```

Comment:
Shouldn't this be called GetLockOwnerName? We don't usually use lower case for a class name.